### PR TITLE
Andrea arsic

### PR DESCRIPTION
**Pull request by AndreaArsic**
I hope that, now, my exercise is done well. I am waiting for your feedback.

### CODE REVIEW

**Review by kristijan-ristic — COMMENTED** [review]
Resolve conflicts that you have with master and I will approve it

File: Candidates.txt [hunk 1]
```diff
@@ -1,3 +1,2 @@
 Bogdan Djuric bogdan-djuric
-Aleksandra your name goes here - replace second line with your name
-Andrea Arsic Andrea-Arsic
\ No newline at end of file
+Andrea Arsic Andrea-Arsic
```
Comment (kristijan-ristic) [inline Candidates.txt:5]: It's not correct. In order to merge it correctly (with latest version of master) you should first pull latest changes from remote master branch to local master branch and than merge it to your branch.

Comment (AndreaArsic) [reply Candidates.txt:5]: I pulled changes on my local branch and I hope it's ok now.

Comment (kristijan-ristic) [reply Candidates.txt:5]: That's fine, but you didn't merge this file as expected. You shouldn't remove some rows from that file, you should just add your at the end of this file (at next empty line)

**Review by kristijan-ristic — APPROVED** [review]

**Pull request merged**